sidestep_engine/data/lyrics_provider_server.py

```
@@ -0,0 +1,298 @@
+"""HTTP lyrics provider backed by a configurable transcription server."""
+
+from __future__ import annotations
+
+import json
+import logging
+import os
+import re
+import uuid
+from typing import Any, Dict, Iterable, List, Tuple
```

Ruff (0.15.5)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)